docs/developing.md

```diff
@@ -266,6 +267,66 @@ To recreate *_vendor_* directory, you can run the following command:
 make vendor
 ```
 
+### Vendoring the OpenShift Installer
+
+For various reasons, Hive vendors the OpenShift Installer. The OpenShift installer brings in quite a few dependencies, so it is important to know the general flow of how to vendor the latest version of the OpenShift Installer.
+
+Things to note:
+* Hive vendors the installer from `@master`, NOT from `@latest`. For go modules, `@latest` means the latest git tag, which for the installer is not up to date.
+* The `go.mod` file contains a section called `replace`. The purpose of this section is to force go modules to use a specific version of that dependency.
+* `replace` directives may need to be copied from the OpenShift Installer or possibly other Hive dependencies. In other words, any dependency may need to be pinned to a specific version.
+* `go mod tidy` is used to add (download) missing dependencies and to remove unused modules.
+* `go mod vendor` is used to copy dependent modules to the vendor directory.
+
+
+The following is a basic flow for vendoring the latest OpenShift Installer. Updating Go modules can sometimes be complex, so it is likely that the flow below will not encompass everything needed to vendor the latest OpenShift Installer. If more steps are needed, please document them here so that the Hive team will know other possible things that need to be done.
+
+Basic flow for vendoring the latest OpenShift Installer:
+
+* Compare the `replace` section of the OpenShift Installer `go.mod` with the `replace` section of the Hive `go.mod`. If the Hive `replace` section has the same module listed as the OpenShift Installer `replace` section, ensure that the Hive version matches the installer version. If it doesn't, change the Hive version to match.
+
+* Edit `go.mod` and change the OpenShift Installer `require` to reference `master` instead of the last version. Go will change it to the version that master points to, so this is a temporary change.
```

Comment:
```suggestion
* Edit `go.mod` and change the OpenShift Installer `require` to reference `master` (or a specific commit hash) instead of the last version. Go will change it to the version that master points to, so this is a temporary change.
```

Reply:
Fixed, thanks!